cmd/root.go

```diff
@@ -130,7 +130,7 @@ func init() {
 	// will be global for your application.
 	// rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.odo.yaml)")
 
-	rootCmd.PersistentFlags().BoolVar(&GlobalConnectionCheck, "skip-connection-check", false, "Skip cluster check")
+	rootCmd.PersistentFlags().BoolVar(&GlobalSkipConnectionCheck, "skip-connection-check", false, "Skip cluster check")
```

Comment:
We are hard coding its value in https://github.com/redhat-developer/odo/pull/821/files#diff-10b3cb428c695c798e01998a4fd2938dR43?
Should we instead only default it to true? or remove this flag altogether?

Reply:
> I am getting this:
> 
> ```
> ✔ ~/go/src/github.com/redhat-developer/odo [pr_821 L|⚑ 59] 
> 12:11 $ odo version
> odo v0.0.13 (HEAD)
> the server could not find the requested resource
> ```

yep this is because the server from which you recently log out, is no more found/ reachable

Reply:
a quick search can help you understand why and when this flag was added ;-) https://github.com/redhat-developer/odo/issues/488

Reply:
> Should we instead only default it to true? 

No it needs to be enabled  by default, otherwise user will get ugly error whithout much explaining what is happening. If I remember correctly this check if users is logged in or not.